lib/nodes/orchestra/locks.cpp

```diff
@@ -0,0 +1,24 @@
+/* Helpers for RTAPI locking primitives
+ *
+ * Author: Steffen Vogel <steffen.vogel@opal-rt.com>
+ *
+ * SPDX-FileCopyrightText: 2025, OPAL-RT Germany GmbH
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mutex>
+
+extern "C" {
+#include <RTAPI.h>
+}
+
+#include <villas/nodes/orchestra/locks.hpp>
+
+static std::mutex globalLock;
```

Comment:
this looks like a connection lock, but is implemented as a global lock. Is this intentional? Can we at least add a note that the lock should in the future be moved to a connection object?